TeeWhyKay/Chomp
# Add edit button for new chomp sess

## Why

No way to edit chomp session that was just created.

## What

Now the admin of the chomp session can edit the details.

### Screenshot

<img width="316" alt="Screenshot 2021-09-23 at 9 51 40 PM" src="https://user-images.githubusercontent.com/11084577/134519552-a031dd5f-7683-4b49-a5c2-ec195a8296de.png">

## How to Test

1. Create session
2. Click edit button
3. Update session

Should get email for creating and updating.
